README.md

```diff
@@ -26,6 +26,25 @@ Run prettier to lint and write - this is also done automatically in a pre-commit
 yarn lint
 ```
 
+## Deploying to remote networks
```

Comment:
The README is primarily public-facing docs. I don't think I'd refer to slack channels here. Primarily provide instructions for general deployments using any unlocked account the user has access to.